Gemfile.lock

```diff
@@ -391,7 +394,7 @@ GEM
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
-    sprockets (4.0.0)
+    sprockets (3.7.2)
```

Comment:
### [sprockets](https://github.com/rails/sprockets)

**Major** version downgrade :chart_with_downwards_trend::exclamation::exclamation: 4.0.0 → 3.7.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **132** commits. See the full changes on [the compare page](https://github.com/rails/sprockets/compare/v4.0.0...v3.7.2).

These are the first 10 commits:
- (0e47b63) [Save String allocations by freezing them](https://github.com/rails/sprockets/commit/0e47b63074ef8fd4078f67d693f13c20e0c309da)
- (8ddce70) [Merge pull request #87 from elia/3-0/let-it-go](https://github.com/rails/sprockets/commit/8ddce70e6f4536e90d78b413392f6db31515ad79)
- (8630ce3) [[close #59] Use relative path for cache keys](https://github.com/rails/sprockets/commit/8630ce365b8c09bc6c409263cee2978658097bca)
- (28c75ae) [Merge pull request #89 from rails/schneems/relative-cache-pa…](https://github.com/rails/sprockets/commit/28c75ae60cc8003741a69b5583d2bb9f4550b200)
- (7d8e9ea) [[ci skip] Version 3.3.0](https://github.com/rails/sprockets/commit/7d8e9eaf962af19bdb7385ace1fb5a0f25cc9b66)
- (326c6df) [Merge pull request #93 from schneems/schneems/3.3.0](https://github.com/rails/sprockets/commit/326c6df01a65970890d2f70110fb885cc69b7b7e)
- (5e5cdc1) [Always pass locals as empty hash](https://github.com/rails/sprockets/commit/5e5cdc1878103c7b3f2ff47352b301f802364cac)
- (1bd5f62) [Release 3.3.1](https://github.com/rails/sprockets/commit/1bd5f62cdb5d06a63ca3f6475c71047925df9c5f)
- (bae6aa7) [Add release tasks](https://github.com/rails/sprockets/commit/bae6aa70ac6bfc8887ad4d1c814881d24fac5c09)
- (b2842d2) [Notify on campfire](https://github.com/rails/sprockets/commit/b2842d2aa226d43941c307234898fd67cfcdaa85)


</details>